src/components/look/look-list/style-tag-list/index.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import { message } from 'antd';
+
+import { Button } from '@src/component/atoms';
+
+import { StyleTag } from '@src/types';
+import styleTagsData from '@src/data/style-tags.json';
+import { LookService } from '@src/services';
+
+type StyleTagListProps = {
+  styleTags: StyleTag[];
+  id: number;
+};
+
+function StyleTagList({ styleTags, id }: StyleTagListProps) {
+  const [tags, setTags] = useState<StyleTag[]>([...styleTags]);
+
+  const handleCheck = (tag: StyleTag) => {
+    tags.find((t) => t.id === tag.id)
+      ? setTags(tags.filter((t) => tag.id !== t.id))
+      : setTags((prevStayleTags) => [...prevStayleTags, tag]);
+  };
+
+  const handleSave = async () => {
+    await LookService.partialUpdate(id, { styleTags: tags });
+    message.success('성공적으로 저장되었습니다.');
+  };
+
+  return (
+    <>
+      <Wrapper>
+        {styleTagsData.map((tag: StyleTag) => (
+          <Button
+            key={tag.id}
+            title={tag.name}
+            onClick={() => handleCheck(tag)}
+            style={{ margin: '0.4rem' }}
+            type={tags.find((tags) => tags.id === tag.id) ? 'primary' : 'line'}
+            size="small"
+            radius="round"
+          />
+        ))}
+      </Wrapper>
+      <Button
+        title="저장"
+        onClick={() => handleSave()}
+        style={{ margin: '1.2rem auto', padding: '0.2rem 2rem' }}
```

Comment:
```suggestion
        style={{ margin: '1.2rem auto', padding: '0.2rem 2rem' }}
```

styled-component로 대체해봐요!

Reply:
66d0864